modules/memfault-firmware-sdk/Kconfig

```diff
@@ -191,6 +191,33 @@ config MEMFAULT_NCS_IMPLEMENT_METRICS_COLLECTION
 	  Implement the Memfault 'memfault_metrics_heartbeat_collect_data()' function
 	  for the selected metrics. Disable this to override the implementation.
 
+config MEMFAULT_NCS_POST_COREDUMP_ON_NETWORK_CONNECTED
+	bool "Post coredump on network connected"
+	depends on PDN
+	depends on LTE_LINK_CONTROL
+	select SMF
+	select SMF_ANCESTOR_SUPPORT
+	select SMF_INITIAL_TRANSITION
+	help
```

Comment:
add a depends on for the supported nrf91 soc Kconfigs

Reply:
Done